src/components/button-swap/button-swap.twig

```diff
@@ -0,0 +1,28 @@
+{#
+  This pattern currently only supports a start icon.
+  End icon support can be added in the future if needed.
+#}
+<section role="region" data-testid="test-id" class="c-button-swap js-c-button-swap{% if class %} {{ class }}{% endif %}">
+  <div class="c-button-swap__subscribe js-c-button-swap__first-btn-wrapper">
+    <span tabindex="-1" role="status" class="u-hidden-visually js-c-button-swap__message">
```

Comment:
I'm curious about the reasoning for adding `tabindex` to these. Could this lead to keyboard users focusing a hidden element they can't see?

Reply:
Ignore this... I was confusing `-1` and `0` tabindexes

Reply:
https://webaim.org/techniques/keyboard/tabindex#negative-one

The -1 value makes it so that you can focus it from javascript, but it is skipped in the natural tab order